content/docs/next/upgrades/upgrade_3_0.md

```diff
@@ -24,7 +24,7 @@ Also, to ensure a smooth rolling upgrade, the running cluster must be healthy. C
 
 Before upgrading etcd, always test the services relying on etcd in a staging environment before deploying the upgrade to the production environment.
 
-Before beginning, [backup the etcd data directory](../../v2/admin_guide.md#backing-up-the-datastore). Should something go wrong with the upgrade, it is possible to use this backup to [downgrade](#downgrade) back to existing etcd version.
+Before beginning, [backup the etcd data directory](../../v2.3/admin_guide.md#backing-up-the-datastore). Should something go wrong with the upgrade, it is possible to use this backup to [downgrade](#downgrade) back to existing etcd version.
```

Comment:
Use `/docs/v2.3/...` instead of a relative URL?
Here and elsewhere.

Reply:
That's a good idea, I do that in other places -- please feel free to ` ```suggest` changes as you're going through this one :)